docs/src/main/paradox/home.md

```diff
@@ -1,54 +1,46 @@
-# Akka Streams Kafka
+# Alpakka Kafka connector
 
-Akka Streams Kafka, also known as Reactive Kafka, is an Akka Streams connector for Apache Kafka.
+The [Alpakka project](https://developer.lightbend.com/docs/alpakka/current/) is an open source initiative to implement stream-aware, reactive, integration pipelines for Java and Scala. It is built on top of [Akka Streams](https://doc.akka.io/docs/akka/current/stream/index.html), and has been designed from the ground up to understand streaming natively and provide a DSL for reactive and stream-oriented programming, with built-in support for backpressure. Akka Streams is a [Reactive Streams](https://www.reactive-streams.org/) and JDK 9+ [java.util.concurrent.Flow](https://docs.oracle.com/javase/10/docs/api/java/util/concurrent/Flow.html)-compliant implementation and therefore [fully interoperable](https://doc.akka.io/docs/akka/current/general/stream/stream-design.html#interoperation-with-other-reactive-streams-implementations) with other implementations.
```

Comment:
"reactive, integration pipelines" - is that an "oxford comma"? Looks strange to me.